spark/v3.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteDataFilesProcedure.java

```diff
@@ -64,11 +64,11 @@ public void testRewriteDataFilesOnPartitionTable() {
     List<Object[]> expectedRecords = currentData();
 
     List<Object[]> output = sql(
-            "CALL %s.system.rewrite_data_files(table => '%s')", catalogName, tableIdent);
+        "CALL %s.system.rewrite_data_files(table => '%s')", catalogName, tableIdent);
```

Comment:
Nit: Usually we try to avoid formatting only changes (even if they are the correct formatting), as the larger diff makes it harder for people that maintain forks to cherry-pick commits. If you aren’t changing the line, I’d suggest leaving it as is.

Reply:
Agree. But I got a specific comment to fix indentation for this file in the spark-3.2 PR. Hence adopting it here also.

But I think checkstyle should catch over indented code in PR, so it can be caught and fixed in the respective PR itself. Let me explore why it is not catching